gcp/observability/logging_test.go

```diff
@@ -306,8 +341,13 @@ func (s) TestServerRPCEventsLogAll(t *testing.T) {
 		},
 		FullDuplexCallF: func(stream grpc_testing.TestService_FullDuplexCallServer) error {
 			for {
```

Comment:
Similar re: not using a loop here.

Reply:
Got rid of every while true loop in all my FullDuplexCallF definitions.